driver/modern_bpf/definitions/missing_definitions.h

```diff
@@ -89,12 +89,28 @@
 #define O_TRUNC 00001000  /* not fcntl */
 #define O_APPEND 00002000
 #define O_NONBLOCK 00004000
-#define O_DSYNC 00010000  /* used to be O_SYNC, see below */
-#define FASYNC 00020000	  /* fcntl, for BSD compatibility */
+#define O_NDELAY O_NONBLOCK
+#define O_DSYNC 00010000 /* used to be O_SYNC, see below */
+#define FASYNC 00020000	 /* fcntl, for BSD compatibility */
+
+#if defined(__TARGET_ARCH_x86)
```

Comment:
You might add an `|| defined(__TARGET_ARCH_s390` here as well. s390x follows the asm-generic/fnctl.h definitions. Thanks.

Reply:
Sure I will do it! Thank you!